torch/csrc/jit/graph_executor.cpp

```diff
@@ -72,6 +73,17 @@ struct ExecutionPlanAutogradFunction : public autograd::Function {
 };
 
 
+// helper to run interpreter on variables until we switch
+// everything to IValue
+inline variable_tensor_list runOneStage(const Code & code, variable_tensor_list inputs) {
+  std::vector<IValue> stack(inputs.begin(), inputs.end());
+  InterpreterState(code).runOneStage(stack);
+  // note: we never unwrapped inputs, because we want autograd to record the trace
```

Comment:
Unwrapped as in not used Variables?

Reply:
Holdover from when we computed on Tensors, I removed it.